### DISCUSSION TITLE
Is there a way to disable calls / TTS?

### DISCUSSION THREAD

**Discussion by c-hoffmann** [open-webui/open-webui#7370] [Q&A]
The title basically says it all. I tried to find an option to disable calls / TTS in the environment variables (.env) and the admin settings in the GUI, but couldn't find it. Can it be disabled for all users?

**Comment by AxENSRennes** [comment]
This is a problem I am also facing. this is a major problem for data privacy reasons. I don't want my data to be sent to exterior TTS providers. And running it locally is computationally expansive and not useful

**Reply by Classic298** [reply]
You can turn it off for users in the admin panel in the group or default user settings, remove the user permission to do calls
